test.json

```diff
@@ -1,18 +1,33 @@
 {
     "resource": "/myresource",
-    "path": "/api/v1/healthz",
-    "httpMethod": "GET",
+    "path": "/api/v1/student",
+    "httpMethod": "PUT",
     "headers": {
         "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8",
         "Accept-Encoding": "gzip, deflate, sdch",
         "Accept-Language": "en-US,en;q=0.8"
     },
     "queryStringParameters": {
         "param1": "value1",
-        "param2": "value2"
+        "param2": "value2", 
+        "uid": {"N": "671901702"}
```

Comment:
Do not use UIN. Use NetID. UIN should be treated as "secret-ish" per uni policy and we are not supposed to ask for UIN.

Reply:
We also have no reason to know their UIN.

Reply:
in the process of updating dynamo